### PR TITLE
Removes ez injectors from basic first aid pouchs

### PR DESCRIPTION
**Pull request by morrowwolf**
# About the pull request

This pr removes the ez injectors from basic first aid kits. They are replaced by the skill required versions.

# Explain why it's good for the game

The line between medic and rifleman is too blurred. As a rifleman you should have to use shittier version autoinjectors to instantly use them (see one use injectors). This will likely not effect gameplay too intensely but should make role skills stand out more.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Morrow
balance: Removes EZ Injectors from basic first aid kits.
balance: Makes first aid pouches empty from non-job specific vendors.
/:cl:

### CODE REVIEW

**Comment by Kuusday** [comment]
Morrow this is a TERRIBLE idea.

Medics will get swamped so hard with an influx of marines begging them to treat their burn and brute damage while they're already trying to stabilize dead/crit patients. Supplies will dwindle even faster making the front basically unsustainable.

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/3042-removes-ez-injectors-from-basic-first-aid-kits/1342/1

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -95,7 +95,7 @@
 		list("Document Pouch", round(scale * 2), /obj/item/storage/pouch/document/small, VENDOR_ITEM_REGULAR),
 		list("Electronics Pouch", round(scale * 2), /obj/item/storage/pouch/electronics, VENDOR_ITEM_REGULAR),
 		list("Explosive Pouch", round(scale * 2), /obj/item/storage/pouch/explosive, VENDOR_ITEM_REGULAR),
-		list("First-Aid Pouch (Full)", round(scale * 5), /obj/item/storage/pouch/firstaid/full, VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Empty)", round(scale * 5), /obj/item/storage/pouch/firstaid, VENDOR_ITEM_REGULAR),
```
Comment (Diegoflores31) [inline code/game/machinery/vending/vendor_types/requisitions.dm:98]: undocumented changes

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 2]
```diff
@@ -240,7 +240,7 @@
 		list("Construction Pouch", round(scale * 2), /obj/item/storage/pouch/construction, VENDOR_ITEM_REGULAR),
 		list("Document Pouch", round(scale * 2), /obj/item/storage/pouch/document/small, VENDOR_ITEM_REGULAR),
 		list("Explosive Pouch", round(scale * 2), /obj/item/storage/pouch/explosive, VENDOR_ITEM_REGULAR),
-		list("First-Aid Pouch (Full)", round(scale * 5), /obj/item/storage/pouch/firstaid/full, VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Empty)", round(scale * 5), /obj/item/storage/pouch/firstaid, VENDOR_ITEM_REGULAR),
```
Comment (Diegoflores31) [inline code/game/machinery/vending/vendor_types/requisitions.dm:243]: undocumented changes as well

**Comment by Diegoflores31** [comment]
also title is a bit confusing you should call it Pouches instead since most people recognize "first aid kit" as the box that HMs get

**Comment by Suchinessa** [comment]
And another one morrondog marinenerf 🤡

**Review by harryob — APPROVED** [review]

**Comment by morrowwolf** [comment]
We'll come back to this one.

**Pull request closed without merging**